tests/out/wgsl/246-collatz-comp.wgsl

```diff
@@ -1,6 +1,6 @@
 [[block]]
 struct PrimeIndices {
-    indices: [[stride(4)]] array<u32>;
+    indices: [[stride(16)]] array<u32>;
```

Comment:
is this expected? an array of `u32` having the stride of 16 now?

Reply:
From my understanding of the spec not defining the layout like what the collatz shader does implies the `shared` layout which essentially means everything is left for the compiler but you cannot discard unused members and the layout must be the same for all shaders with this exact definition.

In Naga's case this means it defaults for `std140` since it follows this rules (`std430` despite generating smaller strides, isn't supported in uniform blocks and there's no logic to pick depending on the block type) and `std140` has this weird rule that the stride must be rounded up to a multiple of the alignment of a `vec4` (16 bytes).